scripts/release-testing/test-e2e-local.js

```diff
@@ -244,11 +244,6 @@ async function testRNTestProject(
   const buildType = 'dry-run';
 
   const reactNativePackagePath = `${REPO_ROOT}/packages/react-native`;
-  const templateRepoFolder = '/tmp/template';
-  const pathToTemplate = path.join(templateRepoFolder, 'template');
-
-  // Cleanup template clone folder
-  exec(`rm -rf ${templateRepoFolder}`);
```

Comment:
Can we keep the cleanup?

Reply:
this folder does not exists anymore. it was pointing to /tmp/template which was the folder where we were cloning the template. We are not cloning the template anymore, so the folder is gone (line 247) and also the cleanup should go at this point.